RecoLocalTracker/SiPhase2VectorHitBuilder/src/VectorHitBuilderAlgorithmBase.cc

```diff
@@ -0,0 +1,151 @@
+#include "RecoLocalTracker/SiPhase2VectorHitBuilder/interface/VectorHitBuilderAlgorithmBase.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "RecoLocalTracker/Records/interface/TkPhase2OTCPERecord.h"
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPE.h"
+
+#include "Geometry/CommonTopologies/interface/PixelTopology.h"
+#include "Geometry/CommonDetUnit/interface/GeomDet.h"
+#include "Geometry/TrackerGeometryBuilder/interface/PixelGeomDetUnit.h"
+
+VectorHitBuilderAlgorithmBase::VectorHitBuilderAlgorithmBase(const edm::ParameterSet& conf) :
+  nMaxVHforeachStack(conf.getParameter<int>("maxVectorHitsinaStack")),
+  barrelCut(conf.getParameter< std::vector< double > >("BarrelCut")),
+  endcapCut(conf.getParameter< std::vector< double > >("EndcapCut")),
+  cpeTag_(conf.getParameter<edm::ESInputTag>("CPE"))
+{}
+
+void VectorHitBuilderAlgorithmBase::initialize(const edm::EventSetup& es)
+{
+  //FIXME:ask Vincenzo
```

Comment:
I took it from [SiStripRecHitConverterAlgorithm](https://github.com/cms-sw/cmssw/blob/master/RecoLocalTracker/SiStripRecHitConverter/src/SiStripRecHitConverterAlgorithm.cc#L27) but I did not manage to understand if needed in my case
@VinInn do you know?

Reply:
it will take a while until one has to manage evolving conditions in phase2...
in any case as long as the init does no computation there is no need to check cache_id

Reply:
Ok, I'll remove the commented code, thanks.